definitions/procedures/installer/run_for_6_11.rb

```diff
@@ -0,0 +1,58 @@
+module Procedures::Installer
+  class RunFor6_11 < ForemanMaintain::Procedure
+    metadata do
+      description 'Ask for Candlepin Database SSL CA certificate'\
+                  'to run installer in post upgrade'
+      param :assumeyes, 'Do not ask for confirmation'
+      manual_detection
+    end
+
+    def run
+      if extdb_and_ssl?
+        run_installer_with_extra_option
+      else
+        run_installer
+      end
+    end
+
+    def ext_db?
+      !feature(:foreman_database).local?
+    end
+
+    def server_db_with_ssl?
+      answers = feature(:installer).answers
+      db_sslmode = answers.fetch('foreman')['db_sslmode']
```

Comment:
I lean towards we should check `answers.fetch('katello')['candlepin_db_ssl']` as for this step we care about Candlepin database configuration the most and this is the leading indicator it's been configured this way.